### PR TITLE
Link CleanRL's PPO Implementation

### PR DESCRIPTION
**Pull request by vwxyzjn**
# What does this PR do?

Hello, this is an awesome repo! We really love using Flax at [CleanRL](https://github.com/vwxyzjn/cleanrl) to seed up our implementations. I was wondering if you'd be interested in linking our PPO implementation, which runs faster (1850% speedup) by using [EnvPool](https://envpool.readthedocs.io/en/latest/), a library for fast parallelizing environments, jitting the entire rollout through [EnvPool's XLA interface](https://envpool.readthedocs.io/en/latest/content/xla_interface.html), storing data more efficiently, and `jax.scan`. Thanks for your consideration!

## Checklist
- [ ] This PR fixes a minor issue (e.g.: typo or small bug) or improves the docs (you can dismiss the other
      checks if that's the case).
- [ ] This change is discussed in a Github issue/
      [discussion](https://github.com/google/flax/discussions) (please add a
      link).
- [ ] The documentation and docstrings adhere to the
      [documentation guidelines](https://github.com/google/flax/blob/main/docs/README.md#how-to-write-code-documentation).
- [ ] This change includes necessary high-coverage tests.
      (No quality testing = no merge!)

### CODE REVIEW

**Comment by marcvanzee** [comment]
Nice! @jheek

**Review by jheek — APPROVED** [review]
Looks great!
